packages/fe-mockserver-core/test/unit/v4/dataAccess.test.ts

```diff
@@ -385,12 +389,16 @@ describe('Data Access', () => {
             )
         );
         expect(actionResult).toBeDefined;
-
+        //expect(actionResult.DraftAdministrativeData).toBeNull();
```

Comment:
What should we do with this commented line?

Reply:
nothing really, i can remove it if you prefer